effekt/shared/src/main/scala/effekt/generator/llvm/LLVM.scala

```diff
@@ -53,7 +53,7 @@ class LLVM extends Compiler[String] {
   // -----------------------------------
   object steps {
     // intermediate steps for VSCode
-    val afterCore = allToCore(Core) map { c => c.main }
+    val afterCore = allToCore(Core) andThen Aggregate andThen core.Optimizer
```

Comment:
I'm a bit unsure about this. While it fixes the issue, I can't use `map { c => c.main }` anymore. Also, was there any reason to exclude the aggregation and optimization phases?

Reply:
Can you please also add polymorphism boxing?